examples/client.rs

```diff
@@ -0,0 +1,55 @@
+use std::{net::SocketAddr, str::FromStr};
+
+#[cfg(target_os = "linux")]
+use wireguard_rs::netlink::{address_interface, create_interface};
+use wireguard_rs::{wgapi::WGApi, Host, IpAddrMask, Key, Peer};
+use x25519_dalek::{EphemeralSecret, PublicKey, StaticSecret};
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    #[cfg(target_os = "linux")]
+    {
+        println!("create interface");
```

Comment:
Maybe move from println to log and configure one stream of info level logs to stdout ?